tools/run-tests.py

```diff
@@ -23,14 +23,14 @@
 import sys
 import settings
 
-if sys.version_info.major >= 3:
-    from runners import util
-else:
-    sys.path.append(os.path.dirname(os.path.realpath(__file__)) + '/runners')
-    import util
+from runners import util
+from runners.util import TERM_NORMAL, TERM_YELLOW, TERM_BLUE, TERM_RED
 
 OUTPUT_DIR = os.path.join(settings.PROJECT_DIR, 'build', 'tests')
 
+# The saved result of get_arguments() for globally access
+options = None # pylint: disable=invalid-name
```

Comment:
This is a valid pylint warning, please don't suppress it. Give this a proper name instead. I am also not sure if we really need to make this whole object global.

Why don't we just do this instead?
```
def iterate_test_runner_jobs(jobs, timeout=None):
```

Reply:
there is to much place to pass options in

Reply:
updated to addressed your concern